### ISSUE TITLE
Update help dialog

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#263]
We have come to a consistent design for the dialog help content in  https://github.com/phetsims/a11y-research/issues/62. We should update the dialog for this sim as well.

**Comment by jessegreenberg** [comment]
Behavior for the appendages is same as a slider, so content will probably be quite similar to the dialog in OL and RIAW.

**Comment by jessegreenberg** [comment]
I updated the dialog so that it has our updated look and feel. We do need some specific content for the arm and leg since it is a new single entry. I took a stab at what this could look like, here it is visually and in the PDOM:

![vis-dialog](https://user-images.githubusercontent.com/6396244/44689142-82837d00-aa24-11e8-94a6-175bc411f583.PNG)

![dialog-invis](https://user-images.githubusercontent.com/6396244/44689111-68499f00-aa24-11e8-908d-014e6469d79c.PNG)

@arouinfar @terracoda does this look OK, or would do you have any thoughts? Ill be sure to make capitalization consistent.

**Comment by arouinfar** [comment]
@jessegreenberg the general structure looks good to me!

I would change the command to "Move arm or leg". I think that "and" could suggest that you are simultaneously moving the arm and leg, while "or" suggests you would be moving one or the other.

**Comment by jessegreenberg** [comment]
Yes, good point! OK sounds good.

**Comment by jessegreenberg** [comment]
Also, BASE just has "Arrow" keys, Ill remove the "up, left, down, and right" in  the PDOM content. Here it is now:
![capture](https://user-images.githubusercontent.com/6396244/44690149-d0e64b00-aa27-11e8-974b-7d14dfb2c11d.PNG)

**Comment by terracoda** [comment]
@jessegreenberg and @arouinfar sorry for the late response. I have a couple concerns:

1. BASE's description is not a good comparison as it is not a slider interaction. It is the 4-way drag. As you know the hand position and leg swing are sliders under the hood. I think _Gravity Force Lab: Basics_ might be a better comparison as the left-right interaction is dominant there as well. 

2. The Up and Down arrows for the Hand position are often not intuitive to visual users in the upper half of the circle. Up Arrow moves clockwise and Down Arrow moves counter-clockwise.

3. The Up Down Arrows on the Leg swing slider, do the opposite as the Hand position, slider. Up moves counter-clockwise and Down moves clockwise.

**Suggested solution:**
- I think it is best to just describe the Left-Right interactions only, even though the Up and Down Arrow keys do indeed work.

Also, we need to keep the names of the sliders in the PDOM in mind as well:
- Hand position, slider
- Leg swing, slider

The original keyboard shortcuts says "move hand or foot", so maybe "Leg swing slider" is clearly indicating that you use it to move the foot. I do not think users have had any trouble making that connection, so let's use "Foot", but I would not use "Arm", just stick with "Hand". It make for a nice parallel.

So here are my suggestions:
**Hand or Foot**
Move hand or foot back and forth       [left arrow icon] [right arrow icon].

In the accessible version it would be:
**Hand or Foot**
Move hand or foot back and forth with left and right arrow keys.

We could easily leave off "back and forth"
**Hand or Foot**
Move hand or foot       [left arrow icon] [right arrow icon].

In the accessible version this would be:
**Hand or Foot**
Move hand or foot with left and right arrow keys.

And of course, I am open to other suggestions if these do not work for folks.

**Comment by terracoda** [comment]
And also, wondering about the "General Navigation" heading. Are we waiting on changing that due to complications with translation? 

We had discussed "Basic Actions" as a new heading in https://github.com/phetsims/a11y-research/issues/113.

**Comment by jessegreenberg** [comment]
Thanks @terracoda! Ill go ahead as you recommend. Reviewing that issue it looks like it was just deferred until it was needed for Friction, Ill go ahead and add the string.

**Comment by terracoda** [comment]
@jessegreenberg, looks good to me. @arouinfar, do you have anything further?

**Comment by arouinfar** [comment]
> I am a fan of the more terse style (without "back and forth")

Agreed @jessegreenberg!

Everything looks good to me @terracoda @jessegreenberg!
